dsa/chapter1/recursion.py

```diff
@@ -0,0 +1,24 @@
+def function():
+
+    # recursive call
+    function()
+
+
+# recursive function
+def offset(x):
```

Comment:
I copied this from the google doc. However, this isn't a recursive function. Should we remove this?